qt/applications/workbench/workbench/plugins/jupyterconsole.py

```diff
@@ -27,7 +27,8 @@
 # from mantidqt.utils.qt import toQSettings when readSettings/writeSettings are implemented
 
 # should we share this with plugins.editor?
-STARTUP_CODE = """from __future__ import (absolute_import, division, print_function, unicode_literals)
+STARTUP_CODE = """
+from __future__ import (absolute_import, division, print_function, unicode_literals)
```

Comment:
sorry for being pedantic here, but is this line still needed?

Reply:
@joseph-torsney Could you take a look at this today? It'll be worth rebasing on master to fix the conflict but maybe hold off until we have done the formatting changes today.

Reply:
this is the only approved PR now, we could as well get this in before the bulk change, see what suits you best

Reply:
Ah sorry just pushed the fix for this. Looks like it was missed in #28334 from when the rest of the `__future__` imports were removed. I can't see a reason why we would need to keep the import in the startup code if it's no longer present in the code editor startup code